scaffoldmaker/utils/interpolation.py

```diff
@@ -787,3 +787,61 @@ def getDoubleCubicHermiteCurvesMidDerivative(ax, ad1, mx, bx, bd1):
     magb = vector.magnitude(bd1)
     magm = arcLengtha + arcLengthb - 0.5*(maga + magb)
     return vector.setMagnitude(md1, magm)
+
+def sampleParameterAlongCenterLine(paramList, lengthList, outputElementsCount):
```

Comment:
Remove 'Center' from name as redundant (I try to avoid using names where there are US spelling differences anyway). I would pass the lengthList as first argument because it is the reference onto which the parameters are mapped (the x to the y).